aiven_mysql_migrate/migration.py

```diff
@@ -160,6 +170,8 @@ def _check_connections(self):
             conn_infos.append(self.target_master)
 
         for conn_info in conn_infos:
+            LOGGER.debug("conn_info :[%s]", conn_info)
```

Comment:
does this dump credentials? If it does, it must be removed.

Reply:
Thanks for catching this!